apollo-client/src/test/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtilTest.java

```diff
@@ -68,7 +66,12 @@ public void registerBeanDefinitionIfNotExistsWithExtPropTest() {
     extraPropertyValues.put(someBeanName, "someProperty");
     assertTrue(BeanRegistrationUtil.registerBeanDefinitionIfNotExists(someRegistry, someBeanName,
         getClass(), extraPropertyValues));
+  }
 
+  @Test
+  public void registerBeanDefinitionIfNotExistsWithoutBeanNameTest() {
+    someRegistry.registerBeanDefinition(BeanDefinition.class.getName(), Mockito.mock(BeanDefinition.class));
+    assertFalse(BeanRegistrationUtil.registerBeanDefinitionIfNotExists(someRegistry, BeanDefinition.class));
```

Comment:
It's better to add a unit test to cover `public static boolean registerBeanDefinitionIfNotExists(BeanDefinitionRegistry registry, Class<?> beanClass, Map<String, Object> extraPropertyValues)`.

Reply:
added